docker/Dockerfile

```diff
@@ -0,0 +1,68 @@
+############################################
+# Dockerfile to run the PCIC data portal   #
+# Based on Ubuntu 16.04                    #
+############################################
+
+FROM ubuntu:16.04
+MAINTAINER Carl Masri <cmasri@uvic.ca>
+
+# Get the system level dependencies
+RUN apt-get update && apt-get install -y \
+    python-dev \
+    python-virtualenv \
```

Comment:
Actually, after further consideration, virtualenv is not really necessary, since we're doing this all in a container, right? May as well just install python-pip and do "system" install inside the container with pip.